app/src/pipelines/generate_referrals/pipeline_wrapper.py

```diff
@@ -48,25 +49,54 @@ def setup(self) -> None:
         pipeline = Pipeline()
         pipeline.add_component("llm", AmazonBedrockChatGenerator(model=model))
 
-        prompt_template = haystack_utils.get_phoenix_prompt("generate_referrals")
+        # set up the pipeline with the default prompt_version_id: latest if local, configuration value if docker/deployed
+        prompt_template = haystack_utils.get_phoenix_prompt(
+            "generate_referrals"
+        )  # TODO MRH replace with component
         pipeline.add_component(
             instance=ChatPromptBuilder(
                 template=prompt_template, required_variables=["query", "supports", "resource_json"]
             ),
             name="prompt_builder",
         )
```

Comment:
I think this code needs to be wrapped into a component that takes the `prompt_version_id` dynamic variable and creates and runs a new instance of `ChatPromptBuilder()`.